auth-api/src/auth_api/config.py

```diff
@@ -59,6 +59,7 @@ class _Config:  # pylint: disable=too-few-public-methods
     PROJECT_ROOT = os.path.abspath(os.path.dirname(__file__))
 
     SECRET_KEY = 'a secret'
+    LD_SDK_KEY = os.getenv('LD_SDK_KEY', None)
```

Comment:
Please change the environment variable name LD_SDK_KEY to AUTH_LD_SDK_KEY. We have multiple keys for different applications.

Reply:
Yup made the change